src/test/test-install.c

```diff
@@ -52,12 +52,12 @@ int main(int argc, char* argv[]) {
 
         log_info("/*** enable **/");
 
-        r = unit_file_enable(UNIT_FILE_SYSTEM, 0, NULL, (char**) files, &changes, &n_changes);
+        r = unit_file_enable(UNIT_FILE_SYSTEM, 0, NULL, (char**) files, &changes, &n_changes, NULL, NULL);
```

Comment:
Wouldn't it be useful to install the callback in a few cases and verify that it is called correctly, at least that it's called at all for the each of the verb types? I think it's fairly easy to forget the check for some verb, or maybe call it twice, and a unittest would make this less likely.